tests/custom_bootstrap.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Sonata\AdminBundle\Tests\App\AppKernel;
+
+$_ENV['KERNEL_CLASS'] = AppKernel::class;
+putenv(sprintf('KERNEL_CLASS=%s', $_ENV['KERNEL_CLASS']));
```

Comment:
Similar to what you did here @VincentLanglet : https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/blob/2.x/tests/custom_bootstrap.php